Sources/Valet/SecureEnclaveValet.swift

```diff
@@ -135,6 +148,9 @@ public final class SecureEnclaveValet: NSObject {
     @objc(containsObjectForKey:)
     public func containsObject(forKey key: String) -> Bool {
```

Comment:
We should note in the header doc that it will return false if the keychain isn't accessible.